gr-video-sdl/grc/video_sdl_sink.block.yml

```diff
@@ -32,9 +32,9 @@ parameters:
     default: '0'
 -   id: num_channels
     label: Channels
-    dtype: enum
+    dtype: int
```

Comment:
This change is sensible, I'm assuming to be able to programmatically change the option in grc, but it would also be nice to expose the enums through the API and use that instead of 1/2/3

Reply:
You can't use an enum for "multiplicity". It just doesn't work.

Reply:
gotcha.